.gitignore

```diff
@@ -40,4 +40,5 @@ yarn-error.log*
 
 # Ignore ALL Typescript files.
 *.tsbuildinfo
-next-env.d.ts
\ No newline at end of file
+next-env.d.ts
+competition.json
```

Comment:
Not sure if we need to add `competition.json` here. This file is going to be kept in `generatedData/` and will probably be called `competitionData2024.json`.